booster/unit-tests/Test/Booster/Pattern/MatchEval.hs

```diff
@@ -145,9 +145,9 @@ varsAndValues =
                 failed (DifferentSorts v d)
         , let v = var "X" someSort
               d = dv someSort ""
-           in test "dv matching a var (on RHS): indeterminate" d v $
-                MatchIndeterminate $
-                    NE.singleton (d, v)
+           in test "dv matching a var (on RHS): fail" d v $
+                MatchFailed $
+                    SubjectVariableMatch d (Variable someSort "X")
```

Comment:
This was the test that ensured `df ~= var` is `Indeterminate` , maybe we can add a link to the old issue

Reply:
done